registrazione-sito-primario-pubblica-amministrazione/inserimento-sito-primario.rst

```diff
@@ -0,0 +1,55 @@
+**1.2 Inserimento Sito Primario**
+=================================
+
+-  Effettuata con successo l’autenticazione al sistema e la conferma
+   della registrazione, l’utente sarà posizionato in automatico nella
+   sezione **Siti Web** dell’applicazione e saranno mostrate le seguenti
+   informazioni:
+
+-  **Pubblica Amministrazione.** L’utente tramite questo campo potrà
+   ricercare la Pubblica Amministrazione di interesse tra quelli censiti
+   sul sistema di **IndicePA.gov**. Impostata la Pubblica
+   Amministrazione di interesse saranno mostrati precaricati e non
+   modificabili i seguenti campi:
+
+-  **URL sito istituzionale**
+
+-  **Responsabile ufficio per la transizione digitale** (in caso di
+   assenza di questo valore il campo sarà mostrato vuoto)
+
+-  **Indirizzo email responsabile ufficio per la transizione digitale**
+   (in caso di assenza di questo valore il campo sarà mostrato vuoto)
+
+|image0|
+
+Figura : Schermata Inserimento Sito Istituzionale
+
+-  L’utente dovrà verificare la correttezza formale dei dati
+   visualizzati. Nel caso in cui i dati sia corretti l’utente dovrà
+   selezionare in forma obbligatoria il flag "**Confermo che i dati sono
+   corretti**" e cliccare sul pulsante **Inserisci Sito** e verrà
+   mostrato all’utente messaggio di conferma di avvenuto inserito
+
+|image1|
```

Comment:
Le figura vanno inserite usando la sintassi prevista dalla guida: https://docs.italia.it/italia/docs-italia/docs-italia-guide/it/bozza/appendice-2/figure.html